Quadriga/src/main/webapp/WEB-INF/tiles-defs.xml

```diff
@@ -688,78 +696,104 @@
-		<put-attribute name="title" value="Quadriga Networks visualize" />
-		<put-attribute name="content"
-			value="/WEB-INF/views/networks/networkvisual.jsp" />
-	</definition> -->
-	<!-- <definition name="auth/networks/networkerror" extends="base.definition.networks">
-		<put-attribute name="title" value="Quadriga Networks" />
-		<put-attribute name="content"
-			value="/WEB-INF/views/networks/networkerror.jsp" />
-	</definition> -->
+	<!-- <definition name="auth/networks/visualize" extends="base.definition.networks"> 
+		<put-attribute name="title" value="Quadriga Networks visualize" /> <put-attribute 
+		name="content" value="/WEB-INF/views/networks/networkvisual.jsp" /> </definition> -->
+	<!-- <definition name="auth/networks/networkerror" extends="base.definition.networks"> 
+		<put-attribute name="title" value="Quadriga Networks" /> <put-attribute name="content" 
+		value="/WEB-INF/views/networks/networkerror.jsp" /> </definition> -->
 	<!-- external website -->
 	<definition name="sites/website" extends="base.definition.public">
-	   <put-attribute name="currentPage" value="home" />
-		<put-attribute name="content" value="/WEB-INF/views/website.jsp"/>
+		<put-attribute name="currentPage" value="home" />
+		<put-attribute name="content" value="/WEB-INF/views/website.jsp" />
 	</definition>
-	
+
 	<definition name="sites/browseNetworks" extends="base.definition.public">
-	   <put-attribute name="currentPage" value="networks" />
-		<put-attribute name="content" value="/WEB-INF/views/browseNetworks.jsp"/>
+		<put-attribute name="currentPage" value="networks" />
+		<put-attribute name="content" value="/WEB-INF/views/browseNetworks.jsp" />
 	</definition>
 
-    <!-- Site search -->
-    <definition name="sites/search" extends="base.definition.public">
-        <put-attribute name="currentPage" value="networksearch" />
-        <put-attribute name="content" value="/WEB-INF/views/networksearch.jsp"/>
-    </definition>
-	
-	<!-- <definition name="sites/networks/visualize" extends="base.definition.networks">
-		<put-attribute name="title" value="Quadriga Networks visualize" />
+
+	<!-- Site search -->
+	<definition name="sites/search" extends="base.definition.public">
+		<put-attribute name="currentPage" value="networksearch" />
+		<put-attribute name="content" value="/WEB-INF/views/networksearch.jsp" />
+	</definition>
+
+
+	<definition name="sites/public/PublicWebsiteAbout" extends="base.definition.public">
+		<put-attribute name="currentPage" value="about" />
+		<put-attribute name="content"
+			value="/WEB-INF/views/public/publicwebsiteabout.jsp" />
+	</definition>
+
+
+	<definition name="auth/saveabout" extends="base.definition">
+		<put-attribute name="currentPage" value="saveabout" />
+		<put-attribute name="content"
+			value="/WEB-INF/views/settings/editAbout.jsp" />
+	</definition>
+
+	<definition name="auth/editabout" extends="base.definition">
+		<put-attribute name="currentPage" value="editabout" />
```

Comment:
This is used when you click on Public Page Settings -> Change About Project details. I have used it to specify which jsp it should refer to when auth/editabout is being returned in the controller. When you are talking about menu item, are you asking me to use tiles only for menu items, or items that can be reused in other pages?